Hymns/Favorites/FavoritesView.swift

```diff
@@ -14,7 +14,10 @@ struct FavoritesView: View {
         VStack {
             CustomTitle(title: "Favorites")
             if self.viewModel.favorites.isEmpty {
-                Text("Tap the ♡ on any hymn to add as a favorite").maxSize()
+                ZStack {
+                    Image("favorites illustration").offset(y: -70)
+                    Text("Tap the ♡ on any hymn to add as a favorite").maxSize().offset(y: 70)
```

Comment:
I think the figma has a different string right

Reply:
Resolved in recent commit